test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -8809,7 +8809,8 @@ def forward(self, input_data, prev_state):
         self.run_test(model, (random_data, empty_tensor),
                       input_names=["data", "state"],
                       dynamic_axes={"state": [0, 1, 2, 3, 4]},
-                      test_with_inputs=[(random_data, random_state)])
+                      test_with_inputs=[(random_data, random_state)],
+                      remained_onnx_input_idx=[1])
```

Comment:
I am confused why only the first input is remaining. Can you please print the ONNX graph?
I expected the model to use the prev_state input.

Reply:
This means `state` is remained but `data` is not remaining
This is because in `check_init` function, we only use the size for `input_data` and it is not dynamic axes. The size is fixed as `(1, 5, 30, 30)` as constant, we don't need `data`.

Reply:
So instead of removing data can we add data to the list of inputs with dynamic axes? So that the main graph body of the test model won't change as much.

Reply:
Added a unit test which makes `input_data` fully dynamic.